packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts

```diff
@@ -1243,6 +1243,84 @@ export const PARAMS_AND_SECRETS_LAMBDA_LAYER_ARNS: { [version: string]: { [arch:
       'us-west-2': 'arn:aws:lambda:us-west-2:345057560386:layer:AWS-Parameters-and-Secrets-Lambda-Extension-Arm64:4',
     },
   },
+  'latest': {
```

Comment:
I would suggest to keep the version logic and to keep the new entry in 1st position: 
```suggestion
  '1.0.104': {
```